launchers/rust-launcher/src/main.rs

```diff
@@ -151,6 +169,35 @@ fn compose_arguments(java_dir: &std::path::PathBuf, original_args: &std::vec::Ve
             os.log("itw-rust-debug: splash excluded");
         }
     }
+
+    let native_debug = is_native_debug_enabled(original_args);
+    let (native_debug_toggle, native_debug_config);
+    if native_debug {
+        let (suspend, port);
+        match get_native_debug_parameter(env::vars().collect::<Vec<_>>(), String::from("ITW_NATIVEDEBUG_SUSPEND")){
+            Some(value) => {
+                suspend = value;
+            }
+            _none => {
+                suspend = String::from("n");
+            }
+        }
```

Comment:
In my opinion `if let` would be easier to read.

Reply:
Btw it is more idiomatic using type `None` when matching value returned from Option instead of default.